migrations/2021-06-21-093252_create_games/up.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE games (
+    id INTEGER PRIMARY KEY
+);
+
+-- Going to create a separate table for players to avoid player_1_id, player_2_id etc columns in future.
+CREATE TABLE players (
+    id INTEGER PRIMARY KEY,
+    game_id INTEGER NOT NULL,
+    user_id INTEGER NOT NULL,
+    won BOOLEAN, -- Can be set per player, so that in future, multiple players (e.g. in a team) can win.
+    elo_change INTEGER,
```

Comment:
some kind of timestamp too possibly?

Reply:
SQLite doesn't support date/times out of the box. You have to just use a string or int for it. So which do you reckon: ISO formatted string or Unix timestamp?

Reply:
unix timestamp please